Tools/Scripts/webkitpy/autoinstalled/buildbot.py

```diff
@@ -26,7 +26,7 @@
 from webkitcorepy import AutoInstall, Package, Version
 from webkitpy.autoinstalled import twisted
 
-AutoInstall.install(Package('attrs', Version(21, 4, 0)))
+AutoInstall.install(Package('attr', Version(21, 3, 0), pypi_name='attrs'))
```

Comment:
Just double-checking this: https://github.com/python-attrs/attrs/blob/main/CHANGELOG.md#backward-incompatible-changes suggests that attrs `21.3.0` is the version which moved to `import attrs`.

Also, this is a downgrade from the current version.